pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -447,7 +447,21 @@ func (r *ReconcileBareMetalHost) actionDeleting(prov provisioner.Provisioner, in
 		info.log.Info("ready to be deleted")
 		return deleteComplete{}
 	}
-
+	// Try to power off host before delete except when it is externally provisioned.
+	if !info.host.Spec.ExternallyProvisioned {
+		provResult, err := prov.PowerOff()
+		switch {
+		case err != nil:
+			log.Error(err, fmt.Sprintf("Likely transient error while trying to Power Off Baremetal Host"))
+		case provResult.ErrorMessage != "":
+			log.Error(err, fmt.Sprintf("Error encountered while trying to Power Off Baremetal Host"))
+		case provResult.Dirty:
+			return actionContinue{provResult.RequeueAfter}
```

Comment:
If we're going to ignore the errors, it would be good for the messages to say so. And maybe not to log at error level since the user won't have a chance to do anything about the problem.

Reply:
We are going to ignore the errors but maybe they are useful to the customer and also might prove to be valuable brea crumbs when we are trying to debug a more complex failure?

Reply:
Done.